### PR TITLE
Feature/pc4 6 MiniCard

### PR DESCRIPTION
**Pull request by Patricia-HL**
CreandoMiniCard

### CODE REVIEW

**Comment by olmedo90** [comment]
Captura de pantalla (809)

No es acorde al figma verificar el ancho y alto de tus componentes.

**Review by Ditmar — COMMENTED** [review]
needs works

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "aphrodite": "^2.4.0",
```
Comment (Ditmar) [inline package.json:13]: remove this

**Review by Ditmar — COMMENTED** [review]
needs work

File: package.json (hunk 1 above)
Comment (Patricia-HL) [reply package.json:13]: image

**Comment by Patricia-HL** [comment]
Buenas Noches Inge Por Favor...
image

**Review by Ditmar — APPROVED** [review]

**Pull request merged**